zerver/lib/html_diff.py

```diff
@@ -42,6 +48,7 @@ def chunkize(text, in_tag):
     return chunks, in_tag
 
 def highlight_chunks(chunks, highlight_func):
+    # type: (List[Tuple[str, str]], Callable) -> str
```

Comment:
you can annotate the Callable in more detail here -- I think it clearly takes a str and returns a str, which would be `Callable[[str], str]`.